migrations/sql/R__0.16.1__CE-365_new_seed_data.sql

```diff
@@ -19,5 +19,7 @@ values
     ('COSPRV&EDU', 'CONTACTLPP', 8, 'Y', CURRENT_USER, CURRENT_TIMESTAMP)
 on conflict do nothing;
 
+--This might not be the right long term decision but it will make it so that it will catch people that forget 
+--to update the code table version
 UPDATE case_management.configuration
-SET configuration_value = 2 WHERE configuration_code = 'CDTABLEVER';
\ No newline at end of file
+SET configuration_value = cast(configuration_value as INTEGER)+1 WHERE configuration_code = 'CDTABLEVER';
```

Comment:
@marqueone-ps Is this intended? So with this line of code, the configuration value will auto +1 in every PR merged even though there are no changes in those code-tables

Reply:
@Scarlett-Truong @marqueone-ps  - We need to agree on a way to manage this, it's more of an issue on the complaint management side, where hardcoded numbers in the script are less than the database (which was updated via webEOC new values)

It's not ideal because it means we are pulling new code tables every time, but the alternative is that I need to go into the database after every migration and increase the number to a proper value.